spbuTest2/main.c

```diff
@@ -0,0 +1,105 @@
+#include <stdio.h>
+#include <malloc.h>
+#include <stdbool.h>
+
+void selectionSort(int originalArray[], int arraySize, int beginning, int ending)
+{
+	int indexOfMinimum = beginning;
+	for (int i = beginning + 2; i <= ending; i += 2)
+	{
+		if (originalArray[i] < originalArray[indexOfMinimum])
+		{
+			indexOfMinimum = i;
+		}
+	}
+
+	int temporaryPlace = originalArray[indexOfMinimum];
+	originalArray[indexOfMinimum] = originalArray[beginning];
+	originalArray[beginning] = temporaryPlace;
+
+	if (arraySize - 4 >= 1)
+	{
+		selectionSort(originalArray, arraySize - 2, beginning + 2, ending);
+
+	}
+}
+
+bool theFirstTest()

```

Comment:
```suggestion
bool theFirstTest(void)
```
Иначе это функция с неопределённым количеством аргументов